src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && <UserInfo user={user} />}
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length !== 0
```

Comment:
```suggestion
      {comments.length
```
it will work this way